resources/build/test/locate_emscripten.tests.sh

```diff
@@ -0,0 +1,212 @@
+#!/usr/bin/env bash
+# Keyman is copyright (C) SIL Global. MIT License.
+#
+# Unit tests for locate_emscripten.inc.sh
+
+shopt -s inherit_errexit
+
+## START STANDARD BUILD SCRIPT INCLUDE
+# adjust relative paths as necessary
+THIS_SCRIPT="$(readlink -f "${BASH_SOURCE[0]}")"
+. "${THIS_SCRIPT%/*}/../../../resources/build/build-utils.sh"
+# END STANDARD BUILD SCRIPT INCLUDE
+
+# shellcheck disable=SC2154
+. "${KEYMAN_ROOT}/resources/locate_emscripten.inc.sh"
+. "${KEYMAN_ROOT}/resources/build/test/testing-framework.inc.sh"
+
+# Mocks
+
+function mocked_builder_die() {
+  echo "DIE: $*" 2>&1
+  exit 99
+}
+
+# Setup/Teardown
+
+function setup_file() {
+  BASE_DIR=$(mktemp -d)
```

Comment:
What folder do these tests run from?  It seems like `BASE_DIR` is significant for them, but so far, I'm not seeing any `cd` commands or similar that would lead the methods being tested to run from its directory.